examples/customresourceinterpreter/apis/workload/v1alpha1/zz_generated.deepcopy.go

```diff
@@ -1,3 +1,4 @@
+//go:build !ignore_autogenerated
```

Comment:
@RainbowMango This line has been added again. :>

Reply:
@XiShanYongYe-Chang @RainbowMango 

`kubernetes/kubernetes` also has the same situation.

https://github.com/kubernetes/api/blob/master/apps/v1/zz_generated.deepcopy.go#L2

Reply:
I'm not saying it's wrong. I've added this before.

Reply:
It's time to figure out why and how this change is introduced.

Reply:
Hi, @carlory, which go version do you use?